sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample8_ModelCompose.md

```diff
@@ -0,0 +1,128 @@
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateFormTrainingClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormTrainingClient(new Uri(endpoint), credential);
+```
+
+## Create a composed model
+In our case, we will be writing an application that collects the expenses a company is making. There are 4 main areas where we get purchase orders from (office supplies, office equipment, furniture, and cleaning supplies). Because each area has its own form with its own structure, we need to train a model per form.
+
+```C# Snippet:FormRecognizerSampleTrainVariousModels
+// For this sample, you can use the training forms found in the `trainingFiles` folder.
+// Upload the forms to your storage container and then generate a container SAS URL.
+// For instructions on setting up forms for training in an Azure Storage Blob Container, see
+// https://docs.microsoft.com/azure/cognitive-services/form-recognizer/build-training-data-set#upload-your-training-data
+
+string purchaseOrderOfficeSuppliesUrl = "<purchaseOrderOfficeSupplies>";
+string purchaseOrderOfficeEquipmentUrl = "<purchaseOrderOfficeEquipment>";
+string purchaseOrderFurnitureUrl = "<purchaseOrderFurniture>";
+string purchaseOrderCleaningSuppliesUrl = "<purchaseOrderCleaningSupplies>";
+
+Response<CustomFormModel> purchaseOrderOfficeSuppliesModel = await client.StartTrainingAsync(new Uri(purchaseOrderOfficeSuppliesUrl), useTrainingLabels: true, new TrainingOptions() { DisplayName = "Purchase order - Office supplies" }).WaitForCompletionAsync();
+Response<CustomFormModel> purchaseOrderOfficeEquipmentModel = await client.StartTrainingAsync(new Uri(purchaseOrderOfficeEquipmentUrl), useTrainingLabels: true, new TrainingOptions() { DisplayName = "Purchase order - Office Equipment" }).WaitForCompletionAsync();
+Response<CustomFormModel> purchaseOrderFurnitureModel = await client.StartTrainingAsync(new Uri(purchaseOrderFurnitureUrl), useTrainingLabels: true, new TrainingOptions() { DisplayName = "Purchase order - Furniture" }).WaitForCompletionAsync();
+Response<CustomFormModel> purchaseOrderCleaningSuppliesModel = await client.StartTrainingAsync(new Uri(purchaseOrderCleaningSuppliesUrl), useTrainingLabels: true, new TrainingOptions() { DisplayName = "Purchase order - Cleaning Supplies" }).WaitForCompletionAsync();
+```
+
+When a purchase order happens, the employee in charge uploads the form to our application. The application then needs to recognize the form to extract the total value of the purchase order. Instead of asking the user to look for the specific `modelId` according to the nature of the form, you can create a composed model that aggregates the previews models, and use that model in `StartRecognizeCustomForms` and let the service decide which model fits best according to the form provided.
```

Comment:
I think?

```suggestion
When a purchase order happens, the employee in charge uploads the form to our application. The application then needs to recognize the form to extract the total value of the purchase order. Instead of asking the user to look for the specific `modelId` according to the nature of the form, you can create a composed model that aggregates the previous models, and use that model in `StartRecognizeCustomForms` and let the service decide which model fits best according to the form provided.
```